packages/http-server/src/__tests__/body-params-validation.spec.ts

```diff
@@ -640,7 +640,7 @@ describe('body params validation', () => {
         });
 
         describe('and size bigger than 10MB', () => {
-          test('returns 422', async () => {
+          test('returns 413', async () => {
```

Comment:
this was checking a 413, so I updated the title to match. 

the rest of the changes in this file are formatting changes